nomos-services/blend/src/core/state.rs

```diff
@@ -21,23 +27,31 @@ mod serde {
         ))]
         unsent_processed_messages: HashSet<ProcessedMessage<BroadcastSettings>>,
         unsent_data_messages: HashSet<EncapsulatedMessageWithVerifiedPublicHeader>,
+        token_collector: SessionBlendingTokenCollector,
```

Comment:
Just a bit clearer?

```suggestion
        current_session_token_collector: SessionBlendingTokenCollector,
```

Reply:
Resolved f0dd563a4084ee5b7a3cfae4b61702cdda63b702, but it'd be more worth reviewing https://github.com/logos-blockchain/logos-blockchain/pull/1989#discussion_r2612789866 than this commit.